recipes-rubygems/rubygems-reek_6.0.6.bb

```diff
@@ -13,12 +13,11 @@ EXTRA_RDEPENDS += "\
 DEPENDS:class-native += "\
     rubygems-kwalify-native \
     rubygems-parser-native \
-    rubygems-psych-native \
     rubygems-rainbow-native \
 "
 
-SRC_URI[md5sum] = "80ce170c086b22b07009256658dd7e36"
-SRC_URI[sha256sum] = "98b83ad7e96015d3db55f6efab795aff45e8eefbcb17db01894e0636e859d3e9"
```

Comment:
potential issue: Use '_append' instead of ' += ' as it overwrites 'EXTRA_RDEPENDS ?= ""